atmat/pubtools/lattice_tools/atreadbeta.m

```diff
@@ -160,7 +160,7 @@
 end
 
 
-evalin('base','global GLOBVAL');
+assignin('base','GLOBVAL',GLOBVAL);
```

Comment:
It is different from the original: here you create a local variable in the base workspace, identical to the global one (bad idea). Declaring the global variable in the base workspace makes it visible, but that's not so useful. The best is to suppress completely these statements (and the similar ones)

Reply:
Original statement made Octave crush. If these are not needed, I will gladly remove them.

Reply:
Go on, remove all the similar lines that you pointed out

Reply:
It broke some code. Particularly, `machine_data`. I'll need some time to fix it